data/db-schema/src/main/resources/net/corda/db/schema/rbac/migration/rest-rbac-creation-v1.0.xml

```diff
@@ -3,15 +3,15 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
 
-    <property name="schema.name" value="RPC_RBAC" global="false"/>
+    <property name="schema.name" value="REST_RBAC" global="false"/>
 
-    <changeSet author="vkolomeyko" id="rpc-perm-creation-v1.0">
+    <changeSet author="vkolomeyko" id="rest-perm-creation-v1.0">
 
         <sql>
-            CREATE SCHEMA IF NOT EXISTS RPC_RBAC;
+            CREATE SCHEMA IF NOT EXISTS REST_RBAC;
```

Comment:
should be just "RBAC" IMO

Reply:
Yes, I agree it should be just `RBAC`. (i.e. without REST prefix)
Since RBAC can be used in other C5 sub-systems.